keps/sig-apps/3017-pod-healthy-policy-for-pdb/kep.yaml

```diff
@@ -24,13 +26,13 @@ replaces:
 
 stage: alpha
 
-latest-milestone: "v1.24"
+latest-milestone: "v1.26"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
-  alpha: "v1.24"
-  beta: "v1.25"
-  stable: "v1.26"
+  alpha: "v1.26"
+  beta: "v1.27"
+  stable: "v1.28"
```

Comment:
Nit: this will most likely be 1.29, since we usually keep a feature 2 releases at beta for sufficient exposure and feedback.